dist/vl-upload.js

```diff
@@ -139,6 +142,31 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
     return document.body.querySelector('#uploadOverlay');
   }
 
+  get _titleSlotElement() {
+    return this._element.querySelector('slot[name=title]');
+  }
+
+  get _titleElement() {
+    return this.uploadElement.querySelector('.vl-upload__element__button__container');
+  }
+
+  set _titleInnerHTML(innerHTML) {
+    this._titleElement.innerHTML = innerHTML;
+  }
+
+  get _subTitleSlotElement() {
+    return this._element.querySelector('slot[name=sub-title]');
+  }
+
+  get _subTitleElement() {
+    return this.uploadElement.querySelector('small');
```

Comment:
Kunnen we hier niet nog iets specifieker zijn dan 'small' te selecteren?

Reply:
ik bekijk het

Reply:
ik denk dat dit niet meer van toepassing is. Als ik in de src/vl-upload.js ga kijken dan zit daar:

![afbeelding](https://user-images.githubusercontent.com/1375279/115231102-75f04d80-a115-11eb-920d-e0d55b20bb09.png)

Nu ik weet niet waarom het nu zo in de dist zit. Zelf genereer ik die eig niet.